docs/contracts.rst

```diff
@@ -1085,8 +1090,13 @@ initialisation code.
 Before the constructor code is executed, state variables are initialised to
 their specified value if you initialise them inline, or zero if you do not.
 
-After the constructor has run, the final code of the contract is returned. The deployment of
+After the constructor has run, the final code of the contract is deployed
+in the blockchain. The deployment of
```

Comment:
Not a native speaker, but I think it should be `deployed to`. In any case, it should be consistent (it's `deployed to` above).